package.json

```diff
@@ -10,17 +10,22 @@
   "scripts": {
     "react-dev": "webpack -d --watch",
     "seed": "mongoimport --db bandland --collection users --file userData.json --jsonArray",
+    "gen-data": "node --max-old-space-size=8192 database/mockDataGen.js",
```

Comment:
I would recommend committing more often. Adding a script could be a separate commit from the data generation you implemented.

Reply:
Good point! I will make to do so in the future.